04/style.css

```diff
@@ -0,0 +1,331 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body{
+    font-family: 'Poppins', sans-serif;
+    overflow: hidden;
+}
+
+ul{
+    list-style: none;
+}
+
+a{
+    text-decoration: none;
+}
+
+header{
+    position: sticky;
+    top: 0px;
+    background-color: #f38dc5;
+    width: 100%;
+    z-index: 1000;
+}
```

Comment:
Zdecydowanie lepiej gdybyś stylowała po klasach (to dobra praktyka), a jeszcze lepiej gdy będzie to zgodne z BEM